client/components/UploadPopUp.jsx

```diff
@@ -0,0 +1,87 @@
+import { gql, useMutation } from "@apollo/client";
+import React, { useEffect, useRef, useState, setState } from "react";
+import { ImExit } from "react-icons/im";
+import {axios} from 'axios';
+import useAuth from "../store/authStore.js";
+
+const UploadPopUp = ({setShowErrorAlert, setErrorMessage, setShowUploadPopUp, refetch}) => {
+
+  const [file, setFile] = useState()
```

Comment:
Set default state for this, don't leave the parameters empty

Reply:
What is the "default" of a file object...?

Reply:
null or undefined